client/src/components/Employees/EachEmployee.js

```diff
@@ -12,8 +12,15 @@ export default function EachEmployee(props) {
         </div>
       </div>
       <div className="lib">
-        <div className="first">{props.firstLibrary}</div>
-        <div className="second">{props.secondLibrary}</div>
+        <div className={props.currentDisplayMoffitt3}>
+          Moffitt 3: {props.moffitt3TrainingLevel}
```

Comment:
For the library tags, Tetsu and I decided to make the display of the training levels into icons, so switch over the "Moffitt 3: 2" display into the star icons.

Reply:
Resolved